src/tools/perf/perftest.c

```diff
@@ -31,9 +31,9 @@
 #include <sys/types.h>
 #include <sys/poll.h>
 #include <locale.h>
-#if HAVE_MPI
+#if defined (HAVE_MPI)
```

Comment:
why need `if defined` for checking single condition?
could use just `ifdef` without brackets for consistency
(same question for other changes in this PR)

Reply:
i think it's because of "elif"

Reply:
right, for single checks used ```ifdef``` construct, for if/else - ```if defined/elif defined```

Reply:
afair, `elif` is valid to use with `ifdef` as well

Reply:
right, this is just styling, nothing else